package/database/sqlinstance/definition.yaml

```diff
@@ -29,6 +29,36 @@ spec:
               parameters:
                 type: object
                 properties:
+                  region:
+                    type: string
+                    description: Amazon AWS region
```

Comment:
Same as above. Please mention the cloud provider documentation URL in the description to make it clear where the region list is coming from.